go/vt/sqlparser/ast.go

```diff
@@ -2782,6 +2782,16 @@ type (
 		Geom     Expr
 	}
 
+	// PointPropertyType is an enum to get the types of geom format functions with possible values: XCordinate YCordinate Latitude Longitude
+
```

Comment:
nit: we do not need a new line here

Reply:
This new line has been removed. Few of my previous PRs introduced a similar newline after comments in ``ast.go``. I have removes these newlines as well.